djstripe/models/payment_methods.py

```diff
@@ -158,6 +160,9 @@ def api_retrieve(self, api_key=None):
             # like an invalid ID error.
             raise InvalidRequestError("No such source: %s" % (self.id), "id")
 
+        # TODO: Need to pass along stripe_account when the following stripe
+        #       python client issue is fixed:
+        #       https://github.com/stripe/stripe-python/issues/609
```

Comment:
Does this mean we need to require ``stripe>=2.36.1`` ?

Reply:
If I change this code to remove the TODO, yes. Would you like me to do that @therefromhere?

Reply:
Is it OK to hold off on that for this OK?  I'm not keen on forcing people to the very latest stripe python version in a 2.1.1 release.

Or does that break things?

Reply:
> It hasn't broken things for me, but I don't have customers stored on connected accounts. I'll need to test if this works when the customers are stored on the connected account.

@ryancausey if you could test it yeah that'd be handy.  Once we've decided how to handle this I'm happy to merge.

Reply:
@therefromhere I tested this using `stripe==2.35.1` and it will pick up the connected account ID from the customer when retrieving the source, allowing it to work as expected.

I'm going to remove the TODO and rely on this behavior so we don't have to pin to `stripe>=2.36.1` in this or a later PR. We will have to make sure that we don't allow `stripe==2.36.0` where that behavior was accidentally removed, but I think another change already did this.